.nvmrc

```diff
@@ -0,0 +1 @@
+20.18.0
```

Comment:
The fact that the node.js version was not specified on the project side was another factor in this problem I think. So I specified v20.18.0, which is the current LTS.